streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -254,6 +265,19 @@ public TopologyTestDriver(final Topology topology,
         this(topology, config, null);
     }
 
+    /**
+     * Create a new test diver instance.
+     * Initialized the internally mocked wall-clock time with {@link System#currentTimeMillis() current system time}.
```

Comment:
This overload takes `initialWallClockTimeMs`, thus this sentence seems to be incorrect? Should go to `TopologyTestDriver(final Topology topology)` instead?

Reply:
fixed the comment.